packages/styles/globals.import.styl

```diff
@@ -4,7 +4,7 @@ body
 
 body
   font-family inherit
-  > div
+  id['__flow-root']
```

Comment:
I was having trouble positioning the toastr since its added to the body. It needs to be more specific. I suppose if it will always be body's first descendant we could use the first-child selector instead of the id.